chromadb/test/property/test_cross_version_persist.py

```diff
@@ -172,7 +176,7 @@ def persist_generated_data_with_old_version(
     coll = api.create_collection(
         name=collection_strategy.name,
         metadata=collection_strategy.metadata,
-        embedding_function=collection_strategy.embedding_function,
+        embedding_function=lambda x: None,
```

Comment:
This makes the tests faster because we pass none across the process boundary because we can't pickle a lambda